web/react/stores/user_store.jsx

```diff
@@ -3,7 +3,6 @@
 
 var AppDispatcher = require('../dispatcher/app_dispatcher.jsx');
 var EventEmitter = require('events').EventEmitter;
-var assign = require('object-assign');
```

Comment:
I just looked it up and Babelify can handle Object.assign so I'm going to log a new ticket to check out if we still need Object.assign and if so we can move to the Babelify implementation instead of importing a separate polyfill.